spec/requests/heartbeat_controller_spec.rb

```diff
@@ -7,5 +7,16 @@
 
       expect(response.body).to eq "PONG"
     end
+
+    context "when service is in maintenance mode" do
+      before do
+        allow(Settings).to receive(:maintenance_mode_enabled).and_return(true)
+      end
+
+      it "returns PONG" do
+        get "/ping"
+        expect(response.body).to eq "PONG"
```

Comment:
Might be worth checking the status here too, since I think that's one of the things the AWS check is looking for

Reply:
I've added a check on the status too.